src/producer/rollup/Frontend.js

```diff
@@ -41,7 +41,7 @@ module.exports = function Rollup ()
 				}),
 				json(),
 				flow({ pretty: true }),
-				// TODO pug
+				pug()
```

Comment:
`pug()` must DRYely share [this settings](https://github.com/StrangeTransistor/metalbox/blob/dced4786dabb16ca8b0f9c47b00af856005a2939/src/producer/Pug.js#L16-L18).